action.yml

```
@@ -18,15 +18,38 @@ inputs:
     description: Set to true in order to enable timestamps on log messages
     required: false
     default: 'true'
-  # TODO(user): Specify your inputs here
-  input-name:
-    description: Some input description
+  repo-org:
+    description: Github Org to use for PR comment
     required: true
+  repo-name:
+    description: Github Repo Name to use for PR comment
+    required: true
+  pr-number:
+    description: PR number to be used for PR comment
+    required: true
+  aws-account-id:
+    description: AWS account ID to use for ECR repo
+    required: true
+  ecr-repo-name:
+    description: AWS ECR Repo Name to use to view scan
+    required: true
+  ecr-repo-tag:
+    description: AWS ECR Repo image tag to use to view scan
+    required: true
+  use-alpha:
+    description: AWS ECR Repo Alpha image
+    required: true
+    default: 'false'
```

Almost all of our ECRs use the alpha channel so set the default to true to save ourselves an input to manage
```suggestion
    default: 'true'
```

---

should now be resolved